docs/can-guides/contribute/including-new-modules.md

```diff
@@ -0,0 +1,111 @@
+@page guides/contributing/include-new-modules Including New Modules
```

Comment:
I tried not to mention "ecosystem" exclusively because it should be helpful for every list.  As a compromise, though, I tried to use examples specific to "ecosystem".